SimCADO/simcado/simulation.py

```diff
@@ -61,22 +66,22 @@ def run(src, mode="wide", cmds=None, opt_train=None, fpa=None,
     """
 
     if cmds is None:
-        cmds = sim.UserCommands()
+        cmds = UserCommands()
```

Comment:
Needs a path to a default.config file, given that UserCommands requires one now

Reply:
Will default.config be of any use? At some point, the user will have to define which instrument and which instrument mode to use. Either an instrument-mode configuration file or a UserCommands instance will be necessary.